### PR TITLE
Convert comments to php docs in some cases

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Convert comments to php docs in some cases
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | Please indicate how to best verify that this PR is correct.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11747)

### CODE REVIEW

**Comment by MathiasReker** [comment]
I would like to work on the docs, so it could be nice to have this merged asap if it is OK :) @PierreRambaud  @jolelievre

**Comment by MathiasReker** [comment]
`@FIXME` = `@todo` ? :trollface:

File: src/PrestaShopBundle/Install/Upgrade.php
Comment (PierreRambaud) [inline src/PrestaShopBundle/Install/Upgrade.php]: Phpcs say you break something here :)

Comment (MathiasReker) [reply src/PrestaShopBundle/Install/Upgrade.php]: Can the order be the issue, since syntax is `@since [version] [<description>]` ?

**Comment by MathiasReker** [comment]
Still a issue with Travis, can you help me? :)

File: classes/Page.php
Comment (jolelievre) [inline classes/Page.php:58]: missing ending .

File: classes/Product.php
Comment (jolelievre) [inline classes/Product.php:4971]: missing ending .

File: classes/Tools.php
Comment (jolelievre) [inline classes/Tools.php:761]: missing ending .

File: classes/module/Module.php
Comment (jolelievre) [inline classes/module/Module.php:1970]: missing ending .

File: classes/module/Module.php
Comment (jolelievre) [inline classes/module/Module.php:2030]: missing ending .

File: classes/module/Module.php
Comment (jolelievre) [inline classes/module/Module.php:2173]: missing ending .

File: classes/module/Module.php
Comment (jolelievre) [inline classes/module/Module.php:2231]: missing ending .

File: classes/module/Module.php
Comment (jolelievre) [inline classes/module/Module.php:2811]: missing ending .

File: classes/order/OrderDetail.php
Comment (jolelievre) [inline classes/order/OrderDetail.php:464]: missing ending .

File: controllers/admin/AdminImportController.php
Comment (jolelievre) [inline controllers/admin/AdminImportController.php:1023]: missing ending .

File: src/Adapter/Module/Tab/ModuleTabRegister.php
Comment (jolelievre) [inline src/Adapter/Module/Tab/ModuleTabRegister.php:278]: missing ending .

File: src/PrestaShopBundle/Install/Upgrade.php
Comment (jolelievre) [inline src/PrestaShopBundle/Install/Upgrade.php:253]: Why not use the `/** */` form here?

File: src/PrestaShopBundle/Install/Upgrade.php
Comment (jolelievre) [inline src/PrestaShopBundle/Install/Upgrade.php:265]: Same question

**Review by jolelievre — CHANGES_REQUESTED** [review]
Thanks again @MathiasReker 
There are a few missing . at the end of PHPDocs
I think you just need to run php-cs-fixer to fix them
And a comment about `src/PrestaShopBundle/Install/Upgrade.php` but this one seems to have caused you trouble

**Comment by MathiasReker** [comment]
@jolelievre thanks for your review. Sorry, I did not explain my PR at all. I aim to fix php docs starting with `//`  and `/* ` in this PR. Once this is merged, I have some really nice improvements for the php docs, your comments are already covered by a script I prepared :) But I need this to be merged first please. I is better to split the PR's up in two.

File: src/PrestaShopBundle/Install/Upgrade.php
Comment (MathiasReker) [reply src/PrestaShopBundle/Install/Upgrade.php:253]: Travis failed. I did not manage to solve the issue.

**Comment by jolelievre** [comment]
Ok let's not block this one for now then!

**Review by jolelievre — APPROVED** [review]
Second approval from @PrestaShop/prestashop-core-developers

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks everyone!

**Comment by MathiasReker** [comment]
Thanks!

**Pull request merged**
